src/Microsoft.VisualStudio.Composition.Analyzers/Microsoft.VisualStudio.Composition.Analyzers.csproj

```diff
@@ -33,6 +33,10 @@
     </EmbeddedResource>
   </ItemGroup>
 
+  <ItemGroup>
+    <None Update="tools\*.ps1" CopyToOutputDirectory="Always" Pack="true" PackagePath="" />
```

Comment:
We don't need to copy the scripts to the output directory at all. And to set it to `Always` breaks the VS up-to-date check. Can you remove that attribute?

Also, aren't these scripts supposed to go into the `tools` folder in the package? But I see you dropping them into the package root directory.

Here is a sample:
https://github.com/microsoft/vs-threading/blob/162a4b621f9ac4e8e1417d448f445dd11cf03e74/src/Microsoft.VisualStudio.Threading.Analyzers.CodeFixes/Microsoft.VisualStudio.Threading.Analyzers.CodeFixes.csproj#L22

Your use of `None` instead of `Content` should be fine though.

Reply:
Updated. Thanks!